linera-views/src/lru_caching.rs

```diff
@@ -95,6 +95,9 @@ where
     K: KeyValueStoreClient + Send + Sync,
 {
     const MAX_CONNECTIONS: usize = K::MAX_CONNECTIONS;
+    // The LRU caching does the high level so the splitting stuff is done
+    // by the container K used in the constructor
```

Comment:
Technically it could wrap a size-limited client as well. I'd just remove this comment, or write something like:

```suggestion
    // The LRU cache does not change the underlying client's size limit.
```

Reply:
Ok.